tools/scripts/stm32.mk

```diff
@@ -0,0 +1,95 @@
+CFLAGS += -std=gnu11 \
+	-g3 \
+	-DUSE_HAL_DRIVER \
+	-DDEBUG \
+	-O0 \
+	-ffunction-sections \
+	-fdata-sections \
+	--specs=nano.specs \
+	-mfpu=fpv4-sp-d16 \
+	-mfloat-abi=hard \
+	-mthumb \
+	-mcpu=cortex-m4
+
+LDFLAGS	= -mcpu=cortex-m4 \
+	--specs=nosys.specs \
+	-Wl,--gc-sections \
+	-static \
+	--specs=nano.specs \
+	-mfpu=fpv4-sp-d16 \
+	-mfloat-abi=hard \
+	-mthumb \
+	-Wl,--start-group \
+	-lc \
+	-lm \
+	-Wl,--end-group
+
+CC = arm-none-eabi-gcc
+AS = arm-none-eabi-gcc
+AR = arm-none-eabi-ar
+
+rwildcard=$(foreach d,$(wildcard $(1:=/*)),$(call rwildcard,$d,$2) $(filter $(subst *,%,$2),$d))
```

Comment:
You can use rwildcard from generic.mk or is this different?

Reply:
didn't see it, will try to use it

Reply:
https://github.com/analogdevicesinc/no-OS/blob/7728d2e1d9653112fcbaefa86e736b65292e900b/tools/scripts/generic.mk#L9

Reply:
Didn't it work?